tests/test_content_smart_proxy.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+
+@pytest.fixture
+def content_smart_proxy(api, organization):
+    smart_proxies = api.resource('smart_proxies').call('index')
+    content_proxy = {}
+
+    for proxy in smart_proxies['results']:
+        for feature in proxy['features']:
+            if feature['name'] == 'Pulp Node':
+                content_proxy = proxy
+                break
+
+    content_proxy_id = content_proxy['id']
+
+    environments = api.resource('capsule_content').call('lifecycle_environments', {
+        'id': content_proxy['id']
```

Comment:
Any reason you don't use `content_proxy_id` here?
```suggestion
        'id': content_proxy_id
```